tests/NLog.UnitTests/Config/ExtensionTests.cs

```diff
@@ -389,6 +389,7 @@ public void Extension_should_be_auto_loaded_when_following_NLog_dll_format()
                 Assert.NotEmpty(fileLocations);
                 Assert.NotNull(fileLocations[0].Key);
                 Assert.NotNull(fileLocations[0].Value); // Primary search location is NLog-assembly
+                Assert.Equal(fileLocations.Length, fileLocations.Select(f => f.Key).Distinct().Count());
```

Comment:
another option is to group-by and check if there aren't duplicate items, isn't?

Reply:
If it is more readable for you, then please change it. Don't have an opinion.